Flutter/pdf-viewer/text-selection.md

```diff
@@ -104,6 +104,49 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## How to get the seleted text lines in the PDF viewer?
+
+Using [getSelectedTextLines](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewerState/getSelectedTextLines.html) method, you can get the selected text lines in the PDF viewer. The following code example explains the same.
```

Comment:
Using the [getSelectedTextLines](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewerState/getSelectedTextLines.html) method, you can get the selected text lines in the PDF viewer. The following code example explains the same.